<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>端口号：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="8">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>建立连接</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>串口未连接</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>串口状态：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>开始采集数据</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="4">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>停止采集数据</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
